### DISCUSSION TITLE
Click and comment to show your love to the dev

### DISCUSSION THREAD

**Discussion by 12th-devs** [zen-browser/desktop#3286] [General]
@mr-cheff this is for you, here is a place to show your love to this awesome dev. Thank you @mr-cheff, we love this project! keep up the hard work!

**Comment by vraravam** [comment]
Yes - really appreciate your dedication and for giving us this awesome customized version of Firefox @mr-cheff !

**Comment by Lippiece** [comment]
I concur. I'm using the Twilight build as my main browser and fully accept potential bugs that might come with it. It's very interesting to see the dynamic in the project, and I really like the ideas implemented. 

Thank you very much for your work, @mr-cheff. And, of course, the [contributors](https://github.com/zen-browser/desktop/graphs/contributors).

I'm not sure about mentioning the dev since we don't write anything useful here though. Please tell if people in this thread should write the feedback without the `@`.

**Comment by mr-cheffy** [comment]
Thanks everyone 😁🥰

**Reply by 12th-devs** [reply]
Thank you for your amazing browser!
